third-party/build.gradle

```diff
@@ -0,0 +1,123 @@
+import java.util.jar.JarFile
+
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+dependencies {
+    api libs.antlr
+    api libs.antlr.runtime
+    api libs.commons.io
+    api libs.jfreechart
+    api libs.osgi.core
+    api libs.osgi.compendium
+    api libs.gurobi
+    api libs.jarh5obj
+    api libs.jarhdf5
+    api libs.jarhdfobj
+    api libs.obgenesis
+    api libs.slf4jnop
+    api libs.jdom
+    api libs.xerces
+
+    //x64 jars
+    api(":calsimgui") //This is version 1.1.2 of CALSIM GUI developed by DWR.
+    api(":coinor") //coinor solver jar, unavailable in maven central
+    api(":com.google.ortools.linearsolver") //linear solver jar, unavailable in maven central
+    api(":COM") //Provides ibm.netrexx.process package, unavailable in maven central
+    api(":dsm2-input-model") //dms2 input model jar, developed by DWR
+    api(":iText") //provides com.lowagie.text package, matching version unavailable in maven central
+    api(":jnios") //unavialable in maven central
+    api(":jpy") //matching jar unavilable in maven central
+    api(":jpython") //matching jar unavilable in maven central
+    api(":misc-1.1") //custom collection of miscellenious packages. Developed by DWR.
+    api(":parser") //custom collection of parser packages. Developed by DWR.
+    api(":sqljdbc4-2.0") //matching jar unavilable in maven central. Newer version available
+    api(":swixml") //local jar contains additional classes compared to 1.5.144 jar on maven central
+    api(":vista") //developed by DWR. Unavailable on maven central
+    api(":xml") //custom collection of sun xml packages
+
+    //dlls
+    api fileTree(dir: 'libs/win-x64', include: ['cb.dll'])
+    api fileTree(dir: 'libs/win-x64', include: ['jep.dll'])
+}
+
+tasks.register('copyDependencies', Copy) {
+    from(configurations.runtimeClasspath)
+    into 'build/tmp/libs'
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+}
+
+tasks.register('prepareBndFile') {
+    dependsOn(tasks.named("copyDependencies"))
+
+    def bndFile = file('build/bnd/bnd.bnd')
+    outputs.file(bndFile)
+    def bndTemplateFile = file('src/main/bnd/bnd.bnd.template')
```

Comment:
I think I missed this in the PR where I added it, but this should be marked as a task Input similar to the outputs.file() line above

Reply:
Added the missing inputs.file(bndTemplateFile) line.